qrec/Model_Semi_aware/find_unobservables.py

```diff
@@ -1,44 +1,86 @@
+"""
+find unobservables
+
+
+"""
+
+
 import os
 import sys
 
 import numpy as np
+import matplotlib.pyplot as plt
+from numpy.random import choice, uniform
 from scipy.optimize import dual_annealing
 
-path = "Model_Semi_aware/"
+PATH = "Model_Semi_aware/"
 sys.path.insert(0, os.getcwd())
-import matplotlib.pyplot as plt
 
-from qrec.utils import p
+
+from .utils import p
 
 
-def Probability(alpha, sign, beta, observations):
+def probability(alpha, signs, betas, observations):
+    """
+    Compute one minus the geometric mean of the probabilities

```

Comment:
This is my guess about what this function does. @dmtomas Is this correct?

Reply:
Es parte de la versión vieja del código que intentaba calcular con Bayes la intensidad, lo que devuelve eso es la 1 - probabilidad de que un estado alpha produzca todas las anteriores observaciones.

Reply:
No me dejan ver los comentarios el resto del código y no sé que tan actualizado quedó (porque después no lo usamos), pero tiene una raíz o un logaritmo en general para que el número no sea tan chiquito y se pueda optimizar